articles/test-runner-plugins.md

```diff
@@ -0,0 +1,236 @@
+<!--
+parent: 'Developer Guide'
+toc: true
+skip : true
+authors:
+    - 'Bertrand Chevrier'
```

Comment:
- the folder should be called *Test Runner Plugins* instead of *test-runner-plugins*. This is a relatively recent change and probably not documented. `parent` (the category under which the article is listed) is generated from the folder name. `parent` as a meta tag however would overwrite the generated parent.
- `toc` means *This article is the table of contents for other articles* and not applicable in this case
- `skip` means *This article is not ready for the hub and should be disregarded*. This is automatically the case as long as it's not on the `master` branch
- `authors` are taken from the commit messages, again the tag overrides that. The Forge articles have all been committed but not written by me which is why this behaviour exists in the first place